### ISSUE TITLE
add the ability to take a screenshot from atom

### ISSUE THREAD

**Issue by devoncarew** [dart-atom/dartlang#659]
I've needed this several times, and had always had to go to the cmd line.

Perhaps add it in the flutter cli tool, and re-expose in atom?

**Comment by sethladd** [comment]
Do you mean, screenshot of the running app on your phone or simulator? If so, should we re-file/move to the Flutter plugin repo?

**Comment by devoncarew** [comment]
Yup, the this should probably be re-filed. Looks like adb has good support for capturing screenshots.
